### ISSUE TITLE
Creating users for internal authentication

### ISSUE THREAD

**Issue by niCSan** [jitsi/docker-jitsi-meet#971]
I would like to have an option to create users per demand to be moderators on the sessions via composer or some args. 

I run jitsi on ECS and the only way to create users is to use a personalized Dockerfile and `ADD` my _USER.dat_ on the `/config/data/meet%2ejitsi/accounts` path. 

The [documentation](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#authentication) indicate that the only way to create users is running inside the container the `prosodyctl` command, but should be a way to create users in the building process.

**Comment by saghul** [comment]
Do you have a suggestion?

**Comment by niCSan** [comment]
> Do you have a suggestion?

Thanks for your answer @saghul 

Maybe some args bypassed to the .env file something like 

`PROSODY_USERn` = "**USER**" 
and
 `PROSODY_PASSWORDn` = "_HashedPasswd_" 

Where n indicates the number of the users created

**Comment by saghul** [comment]
I don't think that can work because we need to know the exact env variables in order to pass them around.

**Comment by niCSan** [comment]
> I don't think that can work because we need to know the exact env variables in order to pass them around.

Or maybe not many users but only one?

**Comment by saghul** [comment]
This could be implemented by using a file which can be mounted into the container. It could be in this format:

`username\tpassword\n`

So each line would contain username and password separated by a TAB. This is easy to parse in bash and create the users on container startup if the file exists.

**Comment by niCSan** [comment]
That would be great!
